### ISSUE TITLE
Ensure supported Java version

### ISSUE THREAD

**Issue by rd-stefan-neuhaus** [sglahn/gradle-dockerfile-plugin#11]
When applying the plugin version 0.6 to my project I get the following error:
```
Cause: org/sglahn/gradle/docker/DockerfilePlugin has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 56.0
```

According to [Java class file](https://en.wikipedia.org/wiki/Java_class_file#General_layout) the major version 57 refers to Java 13. This is kind of strange, as [build.yml](https://github.com/sglahn/gradle-dockerfile-plugin/blob/master/.github/workflows/build.yml) seems to build with Java 14.

Anyway. My suggestions are:
- Build the plugin with a lower byte code level to enhance compatibility.
- Specify this compatibility in the README.

**Comment by stefanneuhaus** [comment]
I verified that builds work with Java 13, while they fail with Java 12 or below.

Note that the build fails during the configuration phase, which renders even dockerbuild-unrelated task (e.g. dependency-check) unusable with Java below version 13.

**Comment by sglahn** [comment]
I have released version 0.7 which is compatible with all java version starting from 8.
